src/iop/demosaic.c

```diff
@@ -5824,6 +5831,12 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_stack_add_named(GTK_STACK(self->widget), box_raw, "raw");
 }
 
+#undef DT_DEMOSAIC_XTRANS
```

Comment:
Not really sure what's the rationale for #undefing any macros at the end of a source file, when there's no risk of accidental usage or clashes later on?

Reply:
I guess tha'ts a memory leftover from my work in FORTH preprocessor stuff. I guess i can remove the #undefs?